src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const isMale = sex === 'm';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${isMale
+            ? (`wife`)
+            : (`husband`)}`}
```

Comment:
yeah, i'd make it a variable as @UmizDemud suggested
```suggestion
          {`${partnerName} is my ${variable}`}
```